src/ForecastView-styles.js

```diff
@@ -17,7 +19,7 @@ export default StyleSheet.create({
     fontSize: 20,
   },
   icon: {
-    fontFamily: 'Weather Icons',
+    fontFamily: (Platform.OS === 'ios') ? 'Weather Icons' : 'weathericons',
```

Comment:
May extract that out as a variable, and check `(Platform.OS === 'android')` as well. Because RN has supported other platforms like Windows and MacOS.